src/index.css

```
@@ -1,16 +1,61 @@
+  }
+
+  .hover\:border-hoosat-teal:hover {
+    border-color: #14B8A6 !important;
+  }
+
+  .bg-hoosat-slate\/50 {
+    background-color: rgba(30, 41, 59, 0.5) !important;
+  }
+}
+
 @import url(https://fonts.googleapis.com/css?family=Roboto+Mono);
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)